examples/tree_2d_dgsem/elixir_advection_callbacks.jl

```diff
@@ -130,6 +130,7 @@ save_solution = SaveSolutionCallback(interval = 100,
                                      save_final_solution = true,
                                      solution_variables = cons2cons)
 
+# Why?
 example_callback = TrixiExtensionExample.ExampleStepCallback(message = "안녕하세요?")
```

Comment:
The git blame shows that you @ranocha are the last person who modified this line of code.

Is there any technical reason that we have to use Korean here? It is not professional at all from my point of view - how could this be approved?

Reply:
Feel free to modify it, e.g.,

```suggestion
example_callback = TrixiExtensionExample.ExampleStepCallback(message = "Initializing callback")
```